source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
```

Comment:
I know it's in line with the existing `MutexUnix` and `MutexWin32`, but these names are funny looking, like `ImageRasterMonochrome` or so.